expression/builtin_convert_charset.go

```diff
@@ -290,7 +290,7 @@ var convertActionMap = map[funcProp][]string{
 		ast.Like, ast.Strcmp,
 		/* regex */
 		ast.Regexp,
-		/* math */
+		/* mathutil */
```

Comment:
Please revert this, since I think it means the expression/builtin_math dependency.